### PR TITLE
Readme: Badges, troubleshooting, other improvements

### PR DESCRIPTION
**Pull request by StephenWeatherford**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,15 +51,16 @@ The `showExplorer` configuration setting controls the visibility of the Explorer
...
 
-![Docker Hub Login](images/dockerHubLogin.gif)
+## Deploying images to Azure App Service
 
-Your user name and password are stored in your operating system credentials vault (for example macOS keychain, Windows Credential Store) so that you don't need to log in every time. You can log out of Docker Hub by right clicking on the Docker Hub label and choosing log out. This will delete the credentials from the OS store.
+With the Docker Explorer you can deploy images from Docker Hub Registries or Azure Container Registries, directly to an Azure App Service instance. This functionality requires installing the [Azure Account](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account) extension having an Azure Subscription. If you do not have an Azure subscription, [sign up today](https://azure.microsoft.com/en-us/free/?b=16.48) for a free 30 day account and get $200 in Azure Credits to try out any combination of Azure services, plus 12 months of popular free services.
```
Comment (PrashanthCorp) [inline README.md]: I expect a merge conflict here. FYI

**Comment by fiveisprime** [comment]
It may be time to create a `docs` directory with detailed troubleshooting and setup guides.

**Comment by StephenWeatherford** [comment]
@fiveisprime How would users access that?

File: README.md [hunk 2]
```diff
@@ -138,6 +141,27 @@ There are a couple of ways you can contribute to this repository:
...
+If you are using an Azure container registry, you will need to get the username and password from Azure by right-clicking on the Azure container registry in the extension and selecting "Browse in the Azure Portal", then selecting the "Access Keys" tab.
+![Getting Azure username and password](images/AzureUsernamePassword.png)
+
+Finally, execute `docker login`, for example:
+
+```bash
+docker login exampleazurecontainerregistry.azurecr.io
+```
+
+and respond with the username and password from Azure.
```
Comment (PrashanthCorp) [inline README.md]: Consider adding:  Note that the username is different from the email id. I've noticed some confusion arising from the docker website allowing one to use their email id in logging in. See [this docker issue](https://github.com/docker/hub-feedback/issues/831#issuecomment-259230362) for this happening.

**Comment by PrashanthCorp** [comment]
@fiveisprime sounds like a good idea, although it isn't so clear to me whether it would be at docs.visualstudio.com, or on our repo.

**Pull request merged**